read-fonts/src/collections.rs

```diff
@@ -0,0 +1,4 @@
+//! data structures useful for font work
+
+pub mod intset;
+pub use intset::IntSet;
```

Comment:
`std` uses `hash_set` and `btree_set` so I think we should follow with `int_set` just for consistency :shrug:

Reply:
oh I went with `intset` _specifically_ because I thought that's what `std` does. 🙄